### ISSUE TITLE
keras.fit() re-initialises the weights

### ISSUE THREAD

**Issue by fasilpv** [fchollet/keras#6293]
I have a model trained using model.fit() and used model.save() to save it on a physical file. Now, I have another data-set on which I want to resume training with the saved model. But, I found that every fit() call is been considered as a fresh training. That means, it is re-initialising the weights which was generated and saved before.

When I called fit() with epochs 0, then I do not see the weight reset problem. But, I definitely want to try with epochs > 0.

Am I missing something here, or is it an issue with Keras.

Keras version: 2.0.3

Thanks.

**Comment by fchollet** [comment]
`fit` does not reinitialize model weights. However, of course, it changes the values of the weights (through the usual gradient descent process on new data).

Try using a lower learning rate, for instance.
